MetaMask/metamask-extension
# Adds "What's New" popup for staking

## **Description**

The Staking team is hoping to add a "What's new" announcement corresponding with the addition of the "Stake" CTA. This PR creates that announcement

This change should please be released at the same time as https://github.com/MetaMask/MetaMask-planning/issues/1781

## **Manual testing steps**

1. Install MM build with this change
2. The Stake announcement should appear, either last on the "What's New" list (after Ledger Live) if you haven't seen previous announcements or as the only item in "What's New" if you have
3. CTA should not click through to anything (just close the notification)

## **Screenshots/Recordings**

### **Before**

No staking announcement

### **After**

<img width="315" alt="Screenshot 2023-12-18 at 11 20 32 AM" src="https://github.com/MetaMask/metamask-extension/assets/15035587/9e76bf22-4b39-447c-96e6-8d590bbedcb5">


## **Pre-merge author checklist**

- [ ] I’ve included tests if applicable _[not applicable]_
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable _[not applicable]_
- [X] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors. _[I think so but please feel free to update the labels or request that I do if not]_

## **Pre-merge reviewer checklist**
